### ISSUE TITLE
[Accessibility: ACC-255.1] Illustrated Message: Provide alternative text for image.

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12616]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
There should be way for Illustrated Message component to add an appropriate alt attribute for images including poster images of videos that do not contain alt attributes. An alt attribute should be a concise and meaningful replacement for the image.
(OR) It should be added by default in the inbuilt svg tag.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Illustrated Message](https://sap.github.io/fundamental-ngx/#/core/illustrated-message)
The ‘No Selected Filter’ svg element does not have a mechanism that allows an accessible name value to be calculated.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/301a3359-c203-4c2b-879b-cee8776797c9">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 the screenshots shows an application. Can you create a stackblitz example so we can provide you with the right support?
